science/lenstool/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
```

Comment:
why did you include the compilers portgroup? It doesn't look like you're actually using any of it.

Reply:
Good point! You want to call the setup procedure.